### PR TITLE
Build break infrastructure issue template

### PR DESCRIPTION
**Pull request by AlitzelMendez**
This PR was originally opened in: https://github.com/dotnet/core-eng/pull/15601
Moving this PR from core-eng to arcade

This is part of the work to achieve: https://github.com/dotnet/core-eng/issues/14182
I am not sure if you are going to be able to see how it works as part of the pr so feel free the fork in which I tested this, not core-eng but well.

Go to the link: https://github.com/AlitzelMendez/build-result-analysis-test/issues
Click on 'New issue'
The template should be in there

### To double check:

* [x] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/arcade/tree/main/Documentation/Validation

### CODE REVIEW

**Review by markwilkie — DISMISSED** [review]
Thanks Ali!

File: .github/ISSUE_TEMPLATE/build-break-infrastructure-issue-template.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Report Infrastructure Issue - Build Break
+description: File an infrastructure issue causing build breaks to be investigated by the engineering services team.
+labels: ["First Responder", "Detected By - Customer", "Known Build Error"]
```
Comment (missymessa) [inline .github/ISSUE_TEMPLATE/build-break-infrastructure-issue-template.yml:3]: Just to make sure, these labels exist in Arcade already?

File: .github/ISSUE_TEMPLATE/build-break-infrastructure-issue-template.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for reporting an infrastructure build break to get investigated by the engineering services team.
```
Comment (missymessa) [inline .github/ISSUE_TEMPLATE/build-break-infrastructure-issue-template.yml:7]:
```suggestion
      value: Thanks for reporting an infrastructure build breaking issue to be investigated by the engineering services team.
```

File: .github/ISSUE_TEMPLATE/build-break-infrastructure-issue-template.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  - type: input
+    id: build
+    attributes:
+      label: Build
+      description: If the issue was opened from the build analysis, this was automatically filled, feel free to update it with a link to any impacted build.
```
Comment (missymessa) [inline .github/ISSUE_TEMPLATE/build-break-infrastructure-issue-template.yml:12]: I think I had feedback on this text in your previous PR, so make sure to apply my suggestions from that one here, too :)

**Review by ulisesh — APPROVED** [review]
LGTM

**Pull request merged**